scripts/ecdsa/mirror/deploy.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# -x echos all lines for debug
+# set -x
+
+export FOUNDRY_DISABLE_NIGHTLY_WARNING=1
+
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+if [ "$DEPLOY_ENV" = "TESTNET" ]; then
+    SOURCE_RPC_URL=${SOURCE_RPC_URL:-}
+else
+    SOURCE_RPC_URL=${SOURCE_RPC_URL:-http://localhost:8545}
+fi
+if [ "$DEPLOY_ENV" = "TESTNET" ]; then
+    MIRROR_RPC_URL=${MIRROR_RPC_URL:-}
+else
+    MIRROR_RPC_URL=${MIRROR_RPC_URL:-http://localhost:8546}
+fi
+
+if [ -z "$SOURCE_RPC_URL" ]; then
+    echo "Error: SOURCE_RPC_URL is not set in the environment variables (tip: grab from .nodes/avs_deploy.json)."
+    exit 1
+fi
+
+if [ -z "$MIRROR_RPC_URL" ]; then
+    echo "Error: MIRROR_RPC_URL is not set in the environment variables (tip: grab from .nodes/avs_deploy.json)."
+    exit 1
+fi
+
+# Set arbitrary location for config
+export WAVS_MIRROR_CONFIG="/wavs/contracts/deployments/wavs-mirror-config.json"
+
+# Read the deployer private key from file
```

Comment:
Do we want the same deployer on the other chain? Or should we pass this in?
I'm not sure here, more a note to self to decide if this is good.